### PR TITLE
removing acoustic last engaged timestamp from models and tests

### PR DESCRIPTION
**Pull request by chelseybeck**
### What's changed
- removed acoustic_last_engaged_at from models and tests - was only needed temporarily and is causing the DAG to wait for unnecessary dependancies `acoustic_external.raw_recipient_raw_v1` & `acoustic_external.raw_recipient_raw_v1`
- removed suppression model + tests - this moved to the [marketing main suppression list](https://github.com/mozilla/bigquery-etl/tree/main/sql/moz-fx-data-shared-prod/marketing_suppression_list_derived/main_suppression_list_v1)

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-3927)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/braze_derived/users_v1/metadata.yaml [hunk 1]
```diff
@@ -1,16 +1,7 @@
...
-labels:
-  incremental: false
-  schedule: daily
-  owner: cbeck
-bigquery:
-  time_partitioning: null
-scheduling:
-  dag_name: bqetl_braze
-  date_partition_parameter: null
+friendly_name : Braze Users description : | -User Data
```
Comment (lelilia) [inline sql/moz-fx-data-shared-prod/braze_derived/users_v1/metadata.yaml:1]: you only need the | if you go to the next line no?

File: sql/moz-fx-data-shared-prod/braze_derived/users_v1/metadata.yaml [hunk 2]
```diff
@@ -1,16 +1,7 @@
...
+FROM
```
Comment (lelilia) [inline sql/moz-fx-data-shared-prod/braze_derived/users_v1/metadata.yaml:2]: ?

**Review by lelilia — APPROVED** [review]

File: sql/moz-fx-data-shared-prod/braze_derived/users_v1/metadata.yaml (hunk 1 above)
Comment (chelseybeck) [reply sql/moz-fx-data-shared-prod/braze_derived/users_v1/metadata.yaml]: this accidentally got in there

File: sql/moz-fx-data-shared-prod/braze_derived/users_v1/metadata.yaml (hunk 2 above)
Comment (chelseybeck) [reply sql/moz-fx-data-shared-prod/braze_derived/users_v1/metadata.yaml]: this accidentally got in there

**Pull request merged**